src/devices/custom_devices_diy.ts

```diff
@@ -1193,6 +1195,18 @@ const definitions: DefinitionWithExtend[] = [
             }),
         ],
     },
+    {
+        zigbeeModel: ['GADOLOCK'],
+        model: 'GADOLOCK',
+        vendor: 'Custom devices (DiY)',
+        description: 'Garage door with lock',
```

Comment:
Are there any instructions available on how to build such lock? This is a requirement for custom devices to be integrated.

Reply:
Thank you for your review.

I have actually a similar DIY device at home that controls an existing garage door motor in addition to 4 electrical latches at each corners. This project is very specific and I have my own external definition. From this point of view, I can say there is no interrest to pollute zhc repo with such device.

I found no existing device made of such lock type with endpoints. Thus I felt it was valuable (ex: to avoid regression bug like issue https://github.com/Koenkk/zigbee2mqtt/issues/24352) to add this represantative `GADOLOCK` model to re-inforce regression test on such device made of several endpoints using state key in toZigbee converter : onOff, windowCovering, lock.

This being said, I understand without any problem that this approach does not fulfill zhc policy and this device can be removed from this PR without any problem. No matter for me 🙂

Reply:
Yes please remove the device from this PR

Reply:
This is understood : I gonna remove this phantom device.

Reply:
Done